lib/twilio/audiohelper.ts

```diff
@@ -661,16 +693,16 @@ class AudioHelper extends EventEmitter {
   }
 
   /**
-   * Destroys processed stream and update references
+   * Destroys the local processed stream and updates references.
    */
-  private _destroyProcessedStream() {
-    if (this._processor && this._processedStream) {
-      this._log.info('destroying processed stream');
-      const processedStream = this._processedStream;
-      this._processedStream.getTracks().forEach(track => track.stop());
-      this._processedStream = null;
-      this._processor.destroyProcessedStream(processedStream);
-      this._audioProcessorEventObserver.emit('destroy');
+  private _destroyLocalProcessedStream() {
+    if (this._localProcessor && this._localProcessedStream) {
+      this._log.info('destroying local processed stream');
+      const localProcessedStream = this._localProcessedStream;
+      this._localProcessedStream.getTracks().forEach(track => track.stop());
+      this._localProcessedStream = null;
+      this._localProcessor.destroyProcessedStream(localProcessedStream);
```

Comment:
We destroy the local processed stream. But where/when do we destroy the remote processed stream? The AudioProcessor.destroyProcessedStream callback will never trigger for the remote processor.